Source/MQTTnet/Client/Options/MqttClientTlsOptions.cs

```diff
@@ -50,8 +50,13 @@ public sealed class MqttClientTlsOptions
 
 #if NET48 || NETCOREAPP3_1_OR_GREATER
         public SslProtocols SslProtocol { get; set; } = SslProtocols.Tls12 | SslProtocols.Tls13;
+
 #else
         public SslProtocols SslProtocol { get; set; } = SslProtocols.Tls12 | (SslProtocols)0x00003000 /*Tls13*/;
 #endif
+
+#if NET7_0_OR_GREATER
+        public string CertificationAuthoritiesFile { get; set; }
```

Comment:
I'd recommend to make this API always visible but throw on the setter for <net7

Reply:
Other properties in this project are usually not available because they are excluded by the compiler. I would like to stick to this approach and avoid changing the strategy here.